### PR TITLE
Modified Gradle file from compile to implementation.

### PR DESCRIPTION
**Pull request by harinadh-dasari**

### CODE REVIEW

**Review by allustag — APPROVED** [review]
Make sure build is success in local as well as in any of the Dev Clusters.

**Review by Haripriya99j — APPROVED** [review]

**Review by mishraa8 — COMMENTED** [review]
added comment

**Comment by lkemmerer** [comment]
What ticket is this part of?

File: build.gradle [hunk 1]
```diff
@@ -87,65 +87,65 @@ configurations {
...
     compile('org.apache.zookeeper:zookeeper:3.8.0')
     compile('org.apache.curator:curator-test:5.3.0')
 
-    compile 'org.apache.commons:commons-lang3:3.7'
-    compile group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
-    compile 'commons-io:commons-io:2.6'
+    implementation'org.apache.commons:commons-lang3:3.7'
+    implementation group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
+    implementation'commons-io:commons-io:2.6'
     compile('com.github.rholder:guava-retrying:2.0.0') {
```
Comment (lkemmerer) [inline build.gradle:136]: Is there a reason this one wasn't changed?

Comment (harinadh-dasari) [reply build.gradle]: Updated compile() to implementation()

**Review by Haripriya99j — APPROVED** [review]

**Review by lkemmerer — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -87,65 +87,65 @@ configurations {
...
+    implementation'org.slf4j:log4j-over-slf4j:1.7.25'
+    implementation'org.slf4j:jcl-over-slf4j:1.7.25'
 
-    compile 'de.danielbechler:java-object-diff:0.94'
+    implementation'de.danielbechler:java-object-diff:0.94'
 
-    compile 'com.amazonaws:aws-java-sdk:1.11.248'
+    implementation'com.amazonaws:aws-java-sdk:1.11.248'
 
     compile('org.apache.curator:curator-recipes:5.3.0')
```
[the remark this answers is not in the document]
Comment (harinadh-dasari) [reply build.gradle]: Modified Abhay

**Review by mishraa8 — APPROVED** [review]
Looks good.

**Pull request merged**